README.md

```diff
@@ -35,10 +36,11 @@ This repository defines `make` targets for uploading three types of credentials
   DEPLOY_ENV=build make ci upload-compose-secrets
   ```
 
-* DeskPro API key: this is to enable the paas-product-page app to create tickets via the DeskPro API.
+* DeskPro API key and ZenDesk token: this is to enable the paas-product-page app to create tickets via the DeskPro API and in ZenDesk.
```

Comment:
Why do we need both? We won't be talking to DeskPro any more after this change?

Reply:
Shall we? I will add other commit.